src/service/templates/models/rethinkdb/templates/service.model.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const rethink = require('rethinkdbdash');
+
+module.exports = function (options) {
+
+  // Create a rethinkdb instance
+  const r = rethink({
+    db: options.db // @slajax todo: add servers to config
+  });
+
+  // <!--- This code can be removed!
+  // All it does is set up your DB table.
+  r.dbList().contains(options.db) // create db if not exists
```

Comment:
Yeah agreed. I'm looking at some ways that we might move this kind of stuff out but we need something to manage async connections and table creations so that the generator "just works".